src/xcode/ENA/ENA.xcodeproj/project.pbxproj

```diff
@@ -7131,14 +7135,14 @@
 			path = ValidationDecorator;
 			sourceTree = "<group>";
 		};
-		8FDB9401261FBA0F00BA8CB1 /* Submisstion */ = {
+		8FDB9401261FBA0F00BA8CB1 /* Submission */ = {
```

Comment:
Not sure if its a good idea to rename it at this moment. If others had created and changed files in it simultaneously it could be very painful to merge.
I would propose to do this renaming at the end.